maskrcnn_benchmark/modeling/rpn/anchor_generator.py

```diff
@@ -77,13 +77,13 @@ def grid_anchors(self, grid_sizes):
         ):
             grid_height, grid_width = size
             device = base_anchors.device
-            shifts_x = torch.arange(
-                0, grid_width * stride, step=stride, dtype=torch.float32, device=device
-            )
-            shifts_y = torch.arange(
-                0, grid_height * stride, step=stride, dtype=torch.float32, device=device
-            )
-            shift_y, shift_x = torch.meshgrid(shifts_y, shifts_x)
+
+            # WORKAROUND: torch.arange export to ONNX not supported yet.
```

Comment:
Is there a PR or so to fix this? I'd aim for the cleanest solution working with master / master + open PRs.